### DISCUSSION TITLE
A question about cuda code

### DISCUSSION THREAD

**Discussion by Mulbetty** [facebookresearch/pytorch3d#1727] [General]
When I read the src code of cuda in the project, I find the atomicMax function declaration in pytorch3d\csrc\pulsar\cuda\commands.h:line145. And in cuda runtime functions, it already has a same one. It’s interesting that there is no error when building the project with enabled CUDA support from source code. In my mind, the two functions will be conflict when building. Does anyone have an idea about it?. Thanks.
![image](https://github.com/facebookresearch/pytorch3d/assets/58506571/0e755c93-3fa6-45d0-a6e7-e3a9e5268894)

**Comment by bottler** [comment]
I think atomicMax is not a keyword in cuda c++, just a library function, so it can be redefined.
